lib/livebook/notebook/explore/intro_to_vega_lite.livemd

```diff
@@ -2,8 +2,8 @@
 
 ```elixir
 Mix.install([
-  {:vega_lite, "~> 0.1.2"},
-  {:kino, "~> 0.5.0"}
+  {:vega_lite, "~> 0.1.4"},
+  {:kino_vega_lite, "~> 0.1.0"}
```

Comment:
I listed both, because this way it's easier to explain what and why we install below.